packages/pytest-simcore/src/pytest_simcore/postgres_service.py

```diff
@@ -170,7 +170,7 @@ def postgres_engine(postgres_dsn: PostgresTestConfig) -> Iterator[sa.engine.Engi
             )
             with engine.connect():
                 print(
-                    f"Connection to {dsn} succeeded [{json_dumps(attempt.retry_state.retry_object.statistics)}]"
+                    f"Connection to {dsn} succeeded [{json.dumps(attempt.retry_state.retry_object.statistics)}]"

```

Comment:
why change this one? this might work but in general json.dumps does not play well with generic objects or pydantic objects right?

Reply:
I wanted to reduce dependencies because I am using this in a library that does not have `servicelib`. 
It is *only used * here to print this particular line which plays well with `json.dumps` so win-win :-)